library/src/main/java/org/jellyfin/apiclient/interaction/BaseApiClient.java

```diff
@@ -279,7 +277,7 @@ public final String GetApiUrl(String handler)
 		return GetApiUrl(handler, new QueryStringDictionary());
 	}
 
-    protected mediabrowser.apiinteraction.http.HttpHeaders HttpHeaders = new HttpHeaders();
+    protected org.jellyfin.apiclient.interaction.http.HttpHeaders HttpHeaders = new HttpHeaders();
```

Comment:
There are a couple declarations with the full name listed out that look like they can be shortened.